index.js

```diff
@@ -167,6 +169,23 @@ bot.on('ready', () => {
             event);
       }
     });
+
+    schedule.scheduleJob('*/5 * * * *', () => {
```

Comment:
Magic number here. I think it would be nice to be able to change it in chat (with mod permissions or something).